### ISSUE TITLE
Identify Input Purpose

### ISSUE THREAD

**Issue by ghost** [Dental-Revenue/Codebase-2-Dev#167]
For users with screen readers, forms should be made more accessible by adding an autofill feature and autofocus.
Autofill - If the user has personal details saved in the browser, the site should fill out as many fields as possible with that data.
Autofocus - The first or primary field in any form should have the text cursor focused automatically on page load.
